io/disk/tiobench.py.data/tiotest.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/tiotest.c b/tiotest.c
```

Comment:
The same story here, do you want to apply them, or is it just from autotest? I mean the compilation and execution works without it. (this one is actually just style fix, the Makefile fix could be serious if one wants to use custom CC)